### PR TITLE
Add rerun subcommand for rerunning jobs

### PR DESCRIPTION
**Pull request by simonbray**
Implements https://github.com/galaxyproject/planemo/issues/1138.

### CODE REVIEW

**Comment by simonbray** [comment]
I tried to reimplement in a simpler, more user-friendly way:
```
planemo rerun --invocation_id ... --history_id ... --job_id ... --job_id ...
```
With one or more arguments. For all of the invocations, histories and jobs specified, all errored, remappable jobs will be rerun.

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 allure-python-commons
 BeautifulSoup4
-bioblend>=0.14.0
+bioblend>=0.16.0
```
Comment (simonbray) [inline requirements.txt:3]: We need the new job filtering which is only available since bioblend 0.16.0.

**Review by jmchilton — APPROVED** [review]
Amazing - very cool!

**Comment by simonbray** [comment]
@jmchilton thanks a lot for the review, thanks @wm75 for your help testing as well. :+1:

**Pull request merged**